roqoqo/tests/integration/operations/three_qubit_gate_operations.rs

```diff
@@ -126,6 +210,9 @@ fn test_three_qubit_gate_unitarity(gate: GateOperation) {
 #[test_case(Operation::from(ControlledControlledPauliZ::new(0, 1, 2)); "ControlledControlledPauliZ")]
 #[test_case(Operation::from(ControlledControlledPhaseShift::new(0, 1, 2, CalculatorFloat::from(0.2))); "ControlledControlledPhaseShift")]
 #[test_case(Operation::from(Toffoli::new(0, 1, 2)); "Toffoli")]
+#[test_case(Operation::from(ControlledSWAP::new(0, 1, 2)); "ControlledSwap")]
+#[test_case(Operation::from(PhaseShiftedControlledControlledZ::new(0, 1, 2, CalculatorFloat::FRAC_PI_2)); "PhaseShiftedControlledControlledZ")]
+#[test_case(Operation::from(PhaseShiftedControlledControlledPhase::new(0, 1, 2, CalculatorFloat::FRAC_PI_2, CalculatorFloat::PI)); "PhaseShiftedControlledControlledPhase")]
 fn test_twoqubitgates_clone(gate1: Operation) {
```

Comment:
Why does the `test_twoqubitgates_clone` tests three qubit gates ?

Reply:
old copy/paste error